lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb

```diff
@@ -226,10 +238,43 @@ def client_rb_content(chef_server_url, node_name)
             https_proxy #{convergence_options[:bootstrap_proxy].inspect}
           EOM
         end
+        if convergence_options[:policy_name]
+          content << <<-EOM
+            # Policyfile Settings:
+            use_policyfile true
+            policy_document_native_api true
+            policy_group "#{convergence_options[:policy_group]}"
+            policy_name "#{convergence_options[:policy_name]}"
+          EOM
```

Comment:
correct me if i'm wrong here, but i think that is already handled globally on line 251 in my branch, 229 in master:

From my PR branch:

https://github.com/double-z/chef-provisioning/blob/zz/policy_support/lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb#L229-L253

From master:

https://github.com/chef/chef-provisioning/blob/master/lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb#L217-L231

if i need more than that lemme know... pretty sure that handles it already, but will test while doing below to be sure.